sdk/servicebus/service-bus/review/service-bus.api.md

```diff
@@ -70,37 +70,40 @@ export interface DeadLetterOptions {
 export { delay }
 
 // @public
-export type DeleteQueueResponse = QueueResponse;
+export type DeleteQueueResponse = EmptyResponse;
```

Comment:
yes

Reply:
@chradek, when you say `earlier types`, did you mean the `QueueResponse`?
If yes, then the earlier type was not really invalid.

As far as I understood from what @ramya0820 shared before is that the service returns the queue object in its response event for Delete operations. Did I get that right @ramya0820 ?

Reply:
@ramya-rao-a  Yes that's what I meant. 

@ramya0820 Can you confirm what @ramya-rao-a said about the delete operations returning the resource in its response? That would seem to mean we should still be aliasing `QueueResponse` instead of `EmptyResponse`.

Reply:
Delete operations don't actually return any entity, that's why updated it to `EmptyResponse`